opentech/apply/funds/models.py

```diff
@@ -39,12 +40,12 @@
 from .admin_forms import WorkflowFormAdminForm
 from .blocks import CustomFormFieldsBlock, MustIncludeFieldBlock, REQUIRED_BLOCK_NAMES
 from .edit_handlers import FilteredFieldPanel, ReadOnlyPanel, ReadOnlyInlinePanel
-from .workflow import SingleStage, DoubleStage, active_statuses, get_review_statuses, review_statuses
+from .workflow import SingleStage, DoubleStage, active_statuses, get_review_statuses, review_statuses, INITAL_STATE, get_stages
 
 
 WORKFLOW_CLASS = {
-    SingleStage.name: SingleStage,
-    DoubleStage.name: DoubleStage,
+    'Request': SingleStage,
+    'Concept & Proposal': DoubleStage,
```

Comment:
Can the workflow name be defined in the workflow.py? The way I see it, it still is part of workflow metadata and should be kept together with the other definitions.